### PR TITLE
Fix Fedora based image

### PR DESCRIPTION
**Pull request by omron93**
SSIA

### CODE REVIEW

**Comment by omron93** [comment]
@pkubatrh Please review and merge.

**Comment by omron93** [comment]
[test][test-openshift]

File: 7-toolchain/Dockerfile.fedora [hunk 1]
```diff
@@ -1,24 +1,23 @@
-FROM fedora:27
-LABEL MAINTAINER Honza Horak <hhorak@redhat.com>
+FROM registry.fedoraproject.org/f27/s2i-core:latest
```
Comment (pkubatrh) [inline 7-toolchain/Dockerfile.fedora:1]: We should probably move this to a later Fedora base if it is already being touched.

**Comment by pkubatrh** [comment]
otherwise lgtm

File: 7-toolchain/Dockerfile.fedora [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
 
 ENV SUMMARY="Fedora variant of Red Hat Developer Toolset's C/C++ Toolchain from Software Collections" \
     DESCRIPTION="Platform for building C and C++ applications" \
-    NAME=perftools \
+    NAME=toolchain \
     VERSION=1 \
     RELEASE=1 \
```
Comment (pkubatrh) [inline 7-toolchain/Dockerfile.fedora]: Maybe remove RELEASE now that is not (afaict) used?

Comment (omron93) [reply 7-toolchain/Dockerfile.fedora]: Fixed, thanks

File: 7-toolchain/Dockerfile.fedora (hunk 1 above)
Comment (omron93) [reply 7-toolchain/Dockerfile.fedora:1]: Fedora 27 is latest release having gcc 7, so we need to keep f27.

**Comment by pkubatrh** [comment]
[test][test-openshift]

**Comment by omron93** [comment]
@pkubatrh For this repo there is still waiting invitation to @centos-ci user, so status from CentOS CI infrastructure can't be displayed in github PR UI.

Before I fixed your comment, CI was passing. Now the infrastructure has problems with allocating new machines, so currently CI fails.

**Comment by omron93** [comment]
CI is now passing, see:
https://ci.centos.org/view/SCLo-images/job/SCLo-container-devtoolset-rh-fedora/10/
https://ci.centos.org/view/SCLo-images/job/SCLo-container-devtoolset-rh/94/
https://ci.centos.org/view/SCLo-images/job/SCLo-container-devtoolset-rh-openshift/4/

**Comment by pkubatrh** [comment]
lgtm, thanks!

**Pull request merged**